### ISSUE TITLE
Netgear R9000 bricks after 4-5-6 newest releases

### ISSUE THREAD

**Issue by Am0rphous** [mirror/dd-wrt#444]
Hi!

I  have a Netgear R9000 router and tried updating to the latest version (06-19-2024-r56941), but the router somehow bricks. Wireless networks wont start and several lights wont come on (probably because services aren't starting correct). Trying to reboot and do several resets (5-7) magically makes the wifi work but reset the whole router back with default dd-wrt settings. I assumed downgrading to earlier firmware would give more stability, but I experience the problems with several releases such as:
- 05-07-2024-r56237
- 05-13-2024-r56359
- 05-14-2024-r56380
- 05-24-2024-r56490
- 06-08-2024-r56721 
- 06-12-2024-r56820
- 06-19-2024-r56941

Somehow I got the 05-02-2024-r56182 to make it work.

With Firmware: v3.0-r56941 std (06/19/24)
Running `dmesg|grep -E 'error.*failed|failed.*error'` gives
````
bq32k: probe of 0-0068 failed with error -5
ath10k_pci 0001:03:00.0: Direct firmware load for ath10k/QCA9984/hw1.0/firmware-6.bin failed with error -2
ath10k_pci 0001:04:00.0: Direct firmware load for ath10k/QCA9984/hw1.0/firmware-6.bin failed with error -2
````
I was able to make the latest release work 1 time and ran `dmesg` and got the output below. Not sure if it is helpful.
````
<removed>
````

Let me know if there is something I can do to help identify the issue.

Best regards

**Comment by kernel-panic69** [comment]
This repository isn't for seeking help, reporting bugs, etc.; submitting patches is welcome, though.

https://github.com/mirror/dd-wrt/blob/master/README.md

But don't open a ticket, open a query in the forum:

https://forum.dd-wrt.com/phpBB2/viewforum.php?f=28

Keep in mind that these devices are known for the radios failing due to excessive heat. So, to rule that out, you must test on stock firmware.  Nobody else is reporting these issues in the forum.

**Comment by BrainSlayer** [comment]
your log shows that all is working and i own a r9000 which is flashed with latest releases which is working too. so i dont see any issue here

**Comment by BrainSlayer** [comment]
DD-WRT v3.0-r57447 std (c) 2024 NewMedia-NET GmbH
Release: 07/17/24
Board: Netgear Nighthawk X10
==========================================================

     ___  ___     _      _____  ______       ____  ___
    / _ \/ _ \___| | /| / / _ \/_  __/ _  __|_  / / _ \
   / // / // /___/ |/ |/ / , _/ / /   | |/ //_ <_/ // /
  /____/____/    |__/|__/_/|_| /_/    |___/____(_)___/

                       DD-WRT v3.0
                   https://www.dd-wrt.com


==========================================================


BusyBox v1.36.1 (2024-07-17 06:29:21 +07) built-in shell (ash)

root@ALPINE:~# softwarerevision
57447
root@ALPINE:~# nvram get DD_BOARD
Netgear Nighthawk X10
root@ALPINE:~#

**Comment by Am0rphous** [comment]
Hi, thanks for reply!
I tried for 4 hours to update to the latest firmware but the router goes into reboot-loop. I see it failes 5 times before everything is reset with default dd-wrt wifi name. This is because the config states if 5 failed boot attempts occurs, then reset. My issue therefore seem to be the backup of the configuration file. Any issues related to importing configuration files?

I successfully installed an old firmware version and also successfully imported my config file. Updating to new firmware seems to crash due to my config, even though it is not that complicated. I consider do a write down of config and setting up things from scratch.

**Comment by BrainSlayer** [comment]
the only advise i can give. try to find out the conflicting setting or use a serial log and send me the output. so i can check whats happening

**Comment by Am0rphous** [comment]
Thanks for tip! I tried reading in the doc for the R9000 but couldn't find anything about serial or console port. But behind the router it states "serial" I assume at the number 1 RJ-45 port. Is that the serial/console port?

**Comment by kernel-panic69** [comment]
No, the serial port is on the mainboard inside the router. You have to disassemble to some degree to get to it:

https://i.imgur.com/cPBEwxy.jpeg

https://forum.dd-wrt.com/phpBB2/viewtopic.php?p=1214461#1214461

**Comment by BrainSlayer** [comment]
closed due inactivity
